content/documentation/PinePhone_Pro/Software/Multi-distribution_image.adoc

```diff
@@ -0,0 +1,291 @@
+EOF
+----
+
+----
+sudo tee /mnt/$PARTNAME/sd/etc/fstab <<EOF
+#<file system>         <dir>      <type> <options>                  <dump> <pass>
+#uncomment for ARCH
+#PARTLABEL=$PARTNAME   /          ext4   rw,relatime                0      1
+
+#uncomment for MANJARO
+#PARTLABEL=MANJARO     /          ext4   defaults                   0      1
+
+#uncomment for MOBIAN
+#PARTLABEL=$PARTNAME   /          ext4   defaults,x-systemd.growfs  0      1
+
+#uncomment for PMOS
+#PARTLABEL=$PARTNAME   /          ext4   defaults                   0      0
+
+#uncomment for UT
+#PARTLABEL=$PARTNAME   /          ext4   defaults                   0      1
+#PARTLABEL=$PARTNAME   /boot      ext4   defaults                   0      2
+#PARTLABEL=$PARTNAME   /userdata  ext4   defaults                   0      2
+EOF
+----
+
+==== Build PostmarketOS image
+
+You can optionally use link:https://wiki.postmarketos.org/wiki/Pmbootstrap[pmbootstrap] to generate the distribution image on your Linux computer, instead of downloading a pre-made image. Make sure you install pmbootstrap before building the image.
+
+Start creating 2 GB empty image file, format and mount it.
+
+----
+sudo su
+dd if=/dev/zero of=postmarketos.img bs=1 count=0 seek=2G status=progress && sync
+mkfs.ext4 postmarketos.img
+losetup -P /dev/loop0 postmarketos.img
+exit
+----
+
+Build the PostmarketOS image via pmbootstrap:
+
+----
+pmbootstrap init
+pmbootstrap status
+pmbootstrap pull
+pmbootstrap install --sdcard=/dev/[LOOP-DEVICE]
+pmbootstrap shutdown
+----
+
+=== Unmount and detach
+
+To unmount and deatch all building images, run:
+
+----
+sudo umount /mnt/$PARTNAME/*
+sudo rm -r /mnt/$PARTNAME
+sudo losetup -D
+----
+
+== Resizing the partitions
+
+On the first boot, if it doesn't happen automatically, you can manually resize each image to fill the entire partition using GParted GUI software or using the CLI:
+
+----
+sudo e2fsck -f /dev/[DEVICE]$PARTNUMBER
+sudo resize2fs /dev/[DEVICE]$PARTNUMBER
+----
+
+Repeat the building process for each needed distribution.
+
+== Usage
+
+According to megi's link:https://xnux.eu/rk2aw[rk2aw info], use the power button and LED feedback to operate the PinePhone Pro:
+
+* Plug in USB power cord. The LED blinks: 0.5s on, 0.5s off. Battery is slowly charging.
+* Press shortly power button. The graphical menu appears, than just select the image to boot from.
+* Press longer power button, LED starts to blinks rapidly. Release the power button, LED blinks N times each second depending on the selected image.
+
+For example:
+
+* LED blinks once each second and 1st image is selected;
+* LED blinks twice each second and 2nd image is seleted;
+* LED blinks triple each second and 3rd image is selected.
+* Press shortly to move to next image.
+* Press longer to boot the selected image.
```

Comment:
(The content above existed before this PR; it's just a convenient opportunity to discuss it.)

The way in which this is written may be confusing to new users. The rk2aw boot selector (i.e. the non-graphical LED-based one) selects a _bootloader image_; it does not select which distro to boot. All distros in the multi-distro image share the same U-Boot, and rk2aw will boot from it by default.

In general, I think this section should be moved to a dedicated rk2aw page. It's the kind of information the user doesn't need to worry about unless they are doing certain kinds of troubleshooting.

Reply:
reg LED, yuo are right. i re-checked megi's site:

" indicator LED will flash N times quickly each second to indicate currently selected menu item.

Menu items have a fixed meaning:

1. Perform a normal rk2aw boot.
2. Boot from a primary bootloader on a boot device rk2aw was installed to (SPI NOR flash or eMMC). SD card boot will be skipped, even if there's a working bootloader there.
3. Boot from a backup bootloader on a boot device rk2aw was installed to.
4. Enter Rockchip mask ROM USB mode."